wp-content/plugins/hurumap-ui/index.php

```diff
@@ -1,7 +1,7 @@
 <?php
 
 /**
- * Plugin Name: HURUmap UI Blocky
+ * Plugin Name: HURUmap UI Blocks
```

Comment:
I think just *HURUmap UI* is good ... we'll add blocks as well as anything else we need (Meta Box, etc.)